### PR TITLE
Fix routing issues - Closes #180

### LINKED ISSUE #180: http://localhost:8080/ is broken
### Expected behaviour
http://localhost:8080/ shows the app
### Actual behaviour
http://localhost:8080/ redirects to http://localhost:8080/#/404 which displays just the LIsk Nano logo.
The url that works is http://localhost:8080/#/, so http://localhost:8080/ should redirect to that
### Steps to reproduce
- open browser
- go to http://localhost:8080/

### PR DESCRIPTION
**Pull request by alihaghighatkhah**
- Redirect unknown routes to login page.
- Set the transactions tab as default tab of the main page.
- Redirect to landing page from login instead of main.

### CODE REVIEW

File: src/app/components/login/login.js [hunk 1]
```diff
@@ -79,7 +78,7 @@ app.component('login', {
       if (passphrase) {
         this.input_passphrase = passphrase;
         if (this.$rootScope.logged === undefined) {
-          this.$timeout(this.passConfirmSubmit.bind(this), 10);
+          // this.$timeout(this.passConfirmSubmit.bind(this), 10);
         }
```
Comment (karmacoma) [inline src/app/components/login/login.js]: Why not remove whole `if` branch?

Comment (slaweet) [inline src/app/components/login/login.js]: I believe it's for development purposes. Just uncomment it and it does autologin.

Comment (slaweet) [inline src/app/components/login/login.js]: I guess it should be described in a comment in the file

Comment (alihaghighatkhah) [inline src/app/components/login/login.js]: Removed it

**Pull request merged**